source/whats-new.txt

```diff
@@ -41,9 +41,31 @@ What's New in 5.0
 
 New features of the 5.0 {+driver-short+} release include:
 
-- Removal of ``Collection.insert()``, ``Collection.update()``, and
-  ``Collection.remove()`` helper methods.
+- Optional installation of ``@aws-sdk/credential-providers`` as a dependency.
 
+  - The driver no longer includes AWS SDK modules by default. If you
+    install the SDK, ``npm`` notifies you if the version of the SDK you
+    installed is incompatible with the driver. When the dependency is
+    installed, the driver uses the AWS SDK itself to manage credentials
+    from the environment. Use the following ``npm`` command to install the SDK:
+
+    .. code-block:: bash
+
+       npm install --save @aws-sdk/credential-providers@3.186.0
+
```

Comment:
[suggestion] I found it a little odd that you tell us what happens if we install the SDK and then tell us how to install the SDK. I'd recommend switching the order of these.

For example:
The driver no longer includes AWS SDK modules by default. Use the following ``npm`` command to install the SDK:
...
``npm`` notifies you if the version...